uktrade/data-hub-frontend
# Pipeline filter by archive

## Description of change

This PR add checkbox to the my pipeline tabs to include archive results in the tabs.  This is state of the checkbox is stored in redux and is persistent  between tabs.
The default option is to hide the archive items, this is done by adding `archived=False` to the endpoint and to show all items we omit the filter.

## Test instructions
Add a company to your pipeline, click edit and repace /edit with /archive once a reason has been entered it should successfully POST to the API and then return you to the pipeline. Your pipeline should not show on the list by default. Clicking the `Show archived projects`  checkbox should make your archived item appear.

There will be styling added to differentiate between archived and non-archived items in future PRs 


## Screenshots
### Before
![image](https://user-images.githubusercontent.com/5575331/85036170-dc19e600-b17b-11ea-9314-ac8056681f74.png)

### After

![image](https://user-images.githubusercontent.com/5575331/85039585-cd353280-b17f-11ea-9c98-464f8825d4ed.png)


## Checklist

[//]: # "When submitting a PR make sure the code review guidelines have been satisfied.
https://github.com/uktrade/data-hub-frontend/blob/master/docs/Code%20review%20guidelines.md"
